exercises/go-counting/go_counting_test.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import gocounting
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
+
+board5x5 = "\n".join([
+    "  B  ",
+    " B B ",
+    "B W B",
+    " W W ",
+    "  W  "
+])
+
+board9x9 = "\n".join([
+    "  B   B  ",
+    "B   B   B",
+    "WBBBWBBBW",
+    "W W W W W",
+    "         ",
+    " W W W W ",
+    "B B   B B",
+    " W BBB W ",
+    "   B B   "
+])
+
+
+class GoCountingTest(unittest.TestCase):
+    def test_5x5_for_black(self):
+        board = gocounting.GoCounting(board5x5)
+        stone, territory = board.territoryFor((0, 1))
+        self.assertEqual(stone, board.black)
```

Comment:
It is better to define constants at either the module level
```Python
BLACK = "B"
WHITE = "W"
NONE = ""

class GoCounting(object):
    ...
```
or in a separate class
```Python
class Player(object):
    BLACK = "B"
    WHITE = "W"
    NONE = ""

class GoCounting(object):
    ...
```

Reply:
You'll need to make the related changes in `go_counting_test.py` as well.